src/db_rows.rs

```diff
@@ -130,6 +130,28 @@ impl NftMetadata {
     // fn from_blob(blob: serde_json::Value) -> Option<Self> {}
 }
 
+#[derive(Clone, diesel::Insertable, diesel::AsChangeset)]
```

Comment:
Why `AsChangeSet`, do we ever plan to update them?

Reply:
I was considering the case of collection reveals but if this isn't a big deal for the near ecosystem I think we can remove this